_posts/2019-04-17-having-a-coffee-with-every-engineer.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: epic
+title: "What I Learned Having a Coffee with Every Engineer"
+date: "2019-04-17"
+author: ash
+categories: [culture, process, people]
+---
+
+In early 2018, I was set to begin my fifth year working at Artsy. Something about my imminent Artsyversary had me
+thinking about my role within the Engineering team. Not my role as an engineer _per se_, but my role as a
+_colleague_. This is the longest I've ever worked for one company, and as Artsy started growing the team last year,
+I wanted to [leverage my impact][lev] as a longtime colleague to help scale its culture.
+
+Artsy collects quarterly, anonymous, company-wide surveys through [Culture Amp][] to determine how everyone is
+doing. These are great for answering quantitative questions about the team, like "how engaged are we on average?",
+and I always check out the breakdown of answers in the Engineering team. But there's something unsatisfying about
+these reports – they're super-valuable, but they feel impersonal to me.
+
+If I wanted to leverage my impact, I needed to play to my strengths and interests. I'm keenly interested in people
+– as individuals – so I decided that the best way for me to contribute to the team was to get to know everyone as
+individuals. To become someone they could talk to. Someone outside the typical manager/employee structure, who
+could use their history at Artsy to answer questions (or at least point them in the right direction).
+
+So, I set off on a project to meet with every member of Artsy's Engineering team for one-on-one. With no explicit
+goals or expectations, but in line with Artsy's [People are Paramount][pap] value, I got to know my colleagues
+better.
+
+<!-- more -->
+
+I didn't want this project to interfere with my product work, so I decided to take things slow. I started with a
+list of every Artsy engineer and then removed anyone I already had a close working relationship with. Next, I
+prioritized the team leads and randomized the rest of the list. I set up a recurring task (yay OmniFocus) to
+schedule a coffee with an engineer every week. When we met, I'd cross them off my list.
+
+That was over a year ago, so what did I learn? Well, a lot, actually! But to get a sense of the impact my project
+had, you'll need some context.
+
+2018 was a complex year for Artsy, where we both gained and lost more engineers than usual. The increase in
+engineering turnover was generally attributable to what I'll call "cultural churn." Scaling a team comes with new
+challenges, and our team leadership tried to meet those challenges with changes to the team's structure. Some of
+these changes turned out to be more costly than we'd hoped. Our team is in a great spot _now_, but we didn't get
```

Comment:
Unsure about this wording. Perhaps “tougher than we’d hoped” is more appropriately?

Reply:
I'll take another pass at this – this phrasing came from my last one-on-one with an Artsy engineer who left last year, but that feeling didn't translate well into the blog post.

Reply:
Yeah, I think you're right – "tougher" is the right word for here.

Reply:
Thanks!